Picker/Picker.js

```diff
@@ -64,21 +75,29 @@ const PickerBase = kind({
 		 * @default 0
 		 * @public
 		 */
-		value: PropTypes.number
+		value: PropTypes.number,
+
+		/**
+		 * Allows picker to continue from the start of the list after it reaches the end and
+		 * vice-versa.
+		 *
+		 * @type {Boolean}
+		 * @default false
+		 * @public
+		 */
+		wrap: PropTypes.bool
 	},
 
 	defaultProps: {
-		value: 0
+		orientation: 'vertical',
```

Comment:
Let's move the definition of propTypes for `orientation` from below to above.

Reply:
Actually, you could move the all propTypes definition below to propTypes block.
We don't use it directly but we do pass the prop with `rest`.